src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -299,6 +299,16 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
     }
   }
 
+  private def findCallKey(call: Call, index: Option[Int]): Option[CallKey] = {
+    executionStore find {
+      case (k,v) => k.scope == call && ((k.index, index) match {
```

Comment:
Right, this is quick and dirty though because it was just to accommodate the current version of findRunnableCall with the DataAccess changes so it compiles. But it will be completely replaced by Scott's version when we merge.